go/libkb/env.go

```diff
@@ -169,7 +169,7 @@ type Env struct {
 	config        ConfigReader
 	homeFinder    HomeFinder
 	writer        ConfigWriter
-	Test          TestParameters
+	Test          *TestParameters
```

Comment:
Because otherwise [this copy](https://github.com/keybase/client/blob/master/go/libkb/test_common.go#L226) makes it so that `tc.Tp` and `tc.G.Tp` are not equal.